src/main/features/core/applicationMenu.js

```diff
@@ -1,6 +1,8 @@
 import { app, Menu, shell } from 'electron';
 import { showDesktopSettings } from './desktopSettings';
 
+const mainWindow = WindowManager.getAll('main')[0];
```

Comment:
There is no guaruntee that this is the same window or that it isn't `null`.  Can we use `WindowManager.getAll` inside the `click` handler and check that it isn't null before using it